src/components/CtaSection/Cta.scss

```diff
@@ -0,0 +1,32 @@
+@use '../../sass/abstracts/variables';
+
+.cta {
+  margin-top: 80px;
+}
+
+.cta_image {
+  width: 100%;
+  margin-bottom: 40px;
+}
+
+.cta_details {
+  align-items: center;
+  text-align: center;
+}
+
+h1 {
+  color: variables.$clr-neutral-900;
+  font-weight: variables.$fw-bold;
+  font-size: variables.$fs-18;
+  line-height: 2.5rem;
+  padding-bottom: variables.$fs-18;
+}
+
+p {
```

Comment:
Same comment as the previous

Reply:
Yes, I noticed that the color of the text in the skill cards was the brown color. I just added the color to the text and that solved it. Not sure if this part of the code actually affects it?